packages/react-components/lib/doors/door-table-datagrid.tsx

```diff
@@ -0,0 +1,152 @@
+        color: theme.palette.action.disabledBackground,
+      };
+      const open = {
+        color: theme.palette.success.main,
+      };
+      const closed = {
+        color: theme.palette.error.main,
+      };
+      const moving = {
+        color: theme.palette.warning.main,
+      };
+
+      switch (params.row.doorState.current_mode.value) {
+        case DoorMode.MODE_OPEN:
+          return open;
+        case DoorMode.MODE_CLOSED:
+          return closed;
+        case DoorMode.MODE_MOVING:
+          return moving;
+        default:
+          return disabled;
+      }
+    }, [params.row.doorState.current_mode.value]);
+
+    return (
+      <Box sx={labelStyle}>
+        <Typography
+          data-testid="door-state"
+          component="p"
+          sx={{
+            fontWeight: 'bold',
+            fontSize: 14,
+          }}
+        >
+          {params.row.doorState ? doorModeToString(params.row.doorState.current_mode.value) : -1}
+        </Typography>
+      </Box>
+    );
+  };
+
+  const OpenCloseButtons = (params: GridCellParams): React.ReactNode => {
+    return (
+      <Box sx={{ margin: 0, padding: 0, paddingRight: 1 }}>
+        <Button
+          variant="contained"
+          size="small"
+          aria-label="open"
+          sx={{ marginRight: 2 }}
+          onClick={params.row.onClickOpen}
+        >
+          Open
+        </Button>
+        <Button
+          variant="contained"
+          size="small"
+          aria-label="close"
+          onClick={params.row.onClickClose}
+        >
+          Close
+        </Button>
+      </Box>
+    );
+  };
+
+  const columns: GridColDef[] = [
+    {
+      field: 'doorName',
+      headerName: 'Name',
+      width: 90,
+      valueGetter: (params: GridValueGetterParams) => params.row.doorName,
+      flex: 1,
+      filterable: true,
+    },
+    {
+      field: 'levelName',
+      headerName: 'Current floor',
+      width: 150,
+      editable: false,
+      valueGetter: (params: GridValueGetterParams) => params.row.levelName,
+      flex: 1,
+      filterable: true,
+    },
+    {
+      field: 'doorType',
+      headerName: 'Type',
+      width: 150,
+      editable: false,
+      valueGetter: (params: GridValueGetterParams) => doorTypeToString(params.row.doorType),
+      flex: 1,
+      filterable: true,
+    },
+    {
+      field: 'doorState',
+      headerName: 'Door state',
+      width: 150,
+      editable: false,
+      flex: 1,
+      renderCell: DoorState,
+      filterable: true,
+    },
+    {
+      field: '-',
+      headerName: '',
+      width: 150,
+      editable: false,
+      renderCell: OpenCloseButtons,
+      flex: 1,
+      filterable: true,
+    },
+  ];
+
+  return (
+    <div style={{ height: '100%', width: '100%' }}>
+      <DataGrid
+        autoHeight={true}
+        getRowId={(l) => l.index}
+        rows={doors}
+        pageSize={5}
+        rowHeight={38}
+        columns={columns}
+        rowsPerPageOptions={[5]}
```

Comment:
```suggestion
        rowsPerPageOptions={[5]}
        localeText={{
          noRowsLabel: 'No doors available.',
        }}
```

Reply:
Just being a little more specific instead of the default `No rows` message from datagrid

Reply:
This looks much better!
Thanks!